parameters/example_model_params_lockdown.json

```diff
@@ -2,6 +2,7 @@
     "populationSize":10000,
     "nHouseholds":3000,
     "nInfections":10,
```

Comment:
I guess this should now be "nInitialInfections"?  (It looks like the test I added for the example parameters is not doing a very good job of testing the example lockdown parameters!)

Reply:
It should be yes - I think the test only does the non-lockdown model check. Should we re-open this PR to fix it or will we just do it in another PR?

Reply:
Please make another PR - It's not hard to do, and reopening PRs is liable to confuse me.  Thanks!